docs/dev.rst

```diff
@@ -594,18 +658,22 @@ verify the session cookie value on each and every request.
 
 
 Auth debugging
```

Comment:
I have no idea if this section is valid anymore. It also pre-dates me and I've never used it.